src/ros/rviz_grid_viewer.h

```diff
@@ -10,11 +11,20 @@
 #include "../core/state_data.h"
 #include "../core/maps/grid_map.h"
 
+/**
+ * \brief This class converts our data into suitable for TF.
```

Comment:
Better: "The class publishes information about robot's map and location in ROS-compatible format so it can be viewed with rviz"